src/services/conversationsService.js

```diff
@@ -334,6 +334,22 @@ const changeReadOnlyState = async function(token, readOnly) {
 	}
 }
 
+/**
+ * Change the converstaion state
+ * @param {string} token The token of the conversation to be modified
+ * @param {int} state The new state to set
+ */
+const changeConversationState = async function(token, state) {
+	try {
+		const response = await axios.put(generateOcsUrl('apps/spreed/api/v3', 2) + `room/${token}/read-only`, {
+			state: state,
+		})
+		return response
+	} catch (error) {
```

Comment:
don't catch in the service.
I know we did this before but want to stop doing this for new calls.
The caller should handle the error accordingly.